packages/devtools_app/test/hover_test.dart

```diff
@@ -6,33 +6,35 @@ import 'package:devtools_app/src/debugger/hover.dart';
 import 'package:flutter/widgets.dart';
 import 'package:test/test.dart';
 
-const _defaultStyle = TextStyle();
 const _textSpan = TextSpan(children: [
   TextSpan(text: 'hello'),
+  TextSpan(text: ' '),
```

Comment:
add some textStyles (e.g. bold or font sizes) now that you support it correctly so that your test verifies you are really doing the right thing.